packages/atomic-angular/projects/atomic-angular/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.33.6](https://github.com/coveo/ui-kit/compare/@coveo/atomic-angular@1.33.5...@coveo/atomic-angular@1.33.6) (2022-11-04)
+
+**Note:** Version bump only for package @coveo/atomic-angular
+
+
+
+
+
```

Comment:
there's a bunch of unrelated changes in the PR 🤔

Reply:
Indeed. I think I ran `npm run build` and then committed the whole change set without combing through it. I'll revert those. But It's weird that this would even happen tbh.

Reply:
nvm it was just that I hadn't pushed the updated parent branch.